lib/v8.js

```diff
@@ -397,6 +397,31 @@ function deserialize(buffer) {
   return der.readValue();
 }
 
+class MemoryProfileWatchdog {
+  #handle;
+  constructor(options) {
+    validateObject(options, 'options');
+    validateUint32(options.interval, 'interval');
+    validateString(options.filename, 'filename');
+    if (options.maxRss) {
+      validateUint32(options.maxRss, 'maxRss');
+    }
+    if (options.maxUsedHeapSize) {
+      validateUint32(options.maxUsedHeapSize, 'maxUsedHeapSize');
+    }
+    assert(options.maxRss > 0 || options.maxUsedHeapSize > 0,
```

Comment:
it's better to throw `ERR_INVALID_ARG_VALUE`

Reply:
Fixed.